### PR TITLE
Mute youtube videos to autoplay on Android

### PR DESCRIPTION
**Pull request by ioanna0**
## What does this change?

In order for embedded videos to autoplay on Android devices
we need to mute the video on start. Implementing this here is
intended only for use in specific US paid content pages.

Another way of doing this would be to modify the `<video>` element
to include `autoplay` and `muted` and this would prevent mutation
of the iFrame `src` property. However, this doesn't seem to be easily
achievable but could be the focus of future improvements.

See https://developers.google.com/web/updates/2016/07/autoplay

Co-authored-by: Josh Buckland <buck06191@gmail.com>

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [x] Yes (please give details)

As explained above

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Review by shtukas — APPROVED** [review]

File: static/src/javascripts/projects/common/modules/atoms/youtube.js [hunk 1]
```diff
@@ -275,21 +277,30 @@ const shouldAutoplay = (atomId: string): boolean => {
...
-    return (
-        ((isVideoArticle() && isInternalReferrer() && isMainVideo()) ||
-            isFront()) &&
-        !isAutoplayBlockingPlatform() &&
-        flashingElementsAllowed()
-    );
+
+    return {
+        autoplay:
+            ((isVideoArticle() && isInternalReferrer() && isMainVideo()) ||
```
Comment (adamnfish) [inline static/src/javascripts/projects/common/modules/atoms/youtube.js]: Be great to see some tests for this logic, because it is pretty hard to parse as a human! I realise this PR is just a change and therefore there may already be some tests, but if this change didn't entail updating a test then at the very least I suspect we need to add a new test case?

Doesn't have to block this release if you're in a hurry, but I'm always more comfortable when these kind of logic knots are well tested!

**Review by adamnfish — APPROVED** [review]
Added a note about tests, but LGTM otherwise

File: static/src/javascripts/projects/common/modules/atoms/youtube.js (hunk 1 above)
Comment (buck06191) [reply static/src/javascripts/projects/common/modules/atoms/youtube.js]: We had the same conversation. We need to rush this out but will raise a card on Trello to add unit tests. This entire module is missing unit testing for most of its logic so it would require a good chunk of work, but it's something I'd want to sort out myself too.
